app/trades.json

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
You'd better create it as list of objects, not with this date formatting using `_`.